lstm_word_segmentation/word_segmenter_cnn.py

```diff
@@ -0,0 +1,596 @@
+from pathlib import Path
+import numpy as np
+import os, json
+import hypertune
+from icu import Char
+from keras.layers import Dense, TimeDistributed, Embedding, Dropout, Input, Conv1D, Maximum
+from tensorflow import keras
+import tensorflow as tf
+from keras.models import Model
+from keras.callbacks import EarlyStopping
+
+from . import constants
+from .helpers import sigmoid, save_training_plot, upload_to_gcs
+from .text_helpers import get_best_data_text, get_lines_of_text, get_segmented_file_in_one_line
+from .accuracy import Accuracy
+from .line import Line
+from .bies import Bies
+from .grapheme_cluster import GraphemeCluster
+from .code_point import CodePoint
+
+class WordSegmenterCNN:
+    """
+    A class that let you make a CNN, train it, and test it.
+    Args:
```

Comment:
There are args you are setting in __init__ that are not defined here (everything after embedding_type). Pls add them too.